docs/components/RPA/getting-started.md

```diff
@@ -0,0 +1,79 @@
+---
+id: getting-started
+title: Getting Started with RPA
+---
+
+### Create your first script
+
+To get started with RPA, you first need to write an RPA script. The Camunda Modeler offers an interface for easy editing and testing of your scripts.
+
+1. **Download the Camunda Modeler**:
+
+   - Download the latest version of the Camunda Modeler from the [Camunda website](https://camunda.com/download/modeler/). As RPA scripts are run locally on your machine, the RPA editor is only available in the Desktop Modeler.
+
+2. **Open the RPA script editor**:
+
+   - Open the Camunda Modeler and navigate to the RPA script editor.
+     ![A screenshot of the RPA Editor in the Camunda Modeler](img/rpa-editor-initial.png)
+
+3. **Start writing your RPA script using RobotFramework**:
+   - Use the interface provided to write your first RPA script. Scripts use the [RobotFramework](https://robotframework.org/) syntax.
+
+### Test your script
+
+Once you have written your script, you can test it on a local RPA worker.
+
+1. **Start the RPA worker**:
+
+   - Download the latest version of the [RPA worker](https://github.com/camunda/rpa-worker/releases).
+   - Create a new file `rpa-worker.properties` in the same directory as the RPA worker. This is the configuration file for this worker.
+   - Add `camunda.client.zeebe.enabled=true` to `rpa-worker.properties`. This disables connection to Zeebe and allows you to test scripts locally.
```

Comment:
Think there's been a mis-paste here. Setting `camunda.client.zeebe.enabled` to `true` would enable the Zeebe connection, not disable it. But it is disabled by default. I would defer talking about configuring the Worker until it is required (in the **Automate Execution** section, where BPMN/Zeebe are being discussed), and instead focus on getting their first scripts running in the Sandbox within minutes without any config burden